CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Adds support for multiple accounts via new commands `login:use`, `login:add` and `login:list`.
```

Comment:
This feels like a feature that could do with (a) documentation in the README and (b) documentation in FireSite

Reply:
cc @rachelsaunders I added a README section, can you take a look?